packages/fondue/components.vite.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'vite';
+
+/**
+ * @type {import('vite').UserConfig}
+ */
+export default defineConfig({
+    build: {
+        lib: {
+            entry: 'src/subpackages/components.ts',
```

Comment:
Could we have a multi entry config instead of 3 different ones?

Reply:
not when in library mode sadly